src/htdocs/js/detail/DetailView.js

```diff
@@ -139,12 +135,48 @@ var DetailView = function (options) {
     return markup;
   };
 
+  _this.getDetailDownload = function () {
+    var markup,
+        url;
+
+    url = 'ftp://hazards.cr.usgs.gov/web/' +
+        'landslides-post-wildfire-debris-flow/fires/' +
+        _this.data.attributes.mapImage;
```

Comment:
Instead of `_this.data.attributes.mapImage` use the method DetailView.getAttribute('mapImage'), this will keep getDetailDownload from breaking if there is no mapImage attribute.